include/crypto/internal/hash.h

```diff
@@ -91,8 +91,6 @@ static inline bool crypto_shash_alg_has_setkey(struct shash_alg *alg)
 	return alg->setkey != shash_no_setkey;
 }
 
-bool crypto_hash_alg_has_setkey(struct hash_alg_common *halg);
```

Comment:
This change should be discarded. dc4e0ab5f774c is already present as a part of 4.4.107 (418e6374e59e5) but git couldn't tell because CAF didn't pick c311cf44ca74 from 4.4.116 (f839a1d54a436).